tests/test_purchase_label.py

```diff
@@ -101,7 +101,7 @@ def test_purchase_label_using_reference_ids(self, api: shippo.Shippo):
         assert shipment.address_return.object_id == address_from.object_id
 
         transaction = api.transactions.create(
-            TransactionCreateRequest(rate=shipment.rates[0].object_id)
+            TransactionCreateRequest(rate=shipment.rates[0].object_id, order=get_order_object_id(api))
```

Comment:
is there a verification on the order here? maybe I'm missing it without looking at the whole file